yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/presentation/controllers/CustomerController.java

```diff
@@ -31,21 +32,28 @@ public CustomerEntity createCustomer(
         return customerService.createCustomer(customerEntity);
     }
 
+    @PostMapping("/addUser")
```

Comment:
No camel case for REST routes,
addUser isn't clear. Maybe use something like "/add-customer-right" ?